penn_near_east_config.rb

```diff
@@ -18,35 +16,42 @@
 
 to_field 'agg_provider', provider
 
-# MET Museum
+# CHO required
 to_field 'id', normalize_prefixed_id('emuIRN')
+# Skip over missing titles, revisit if data changes
+each_record do |record, context|
+  context.skip!('Missing title') if record['object_name'].nil?
+end
+to_field 'cho_title', column('object_name', split: '|')
 
-to_field 'cho_provenance', column('accession_credit_line')
-to_field 'cho_creator', column('creator')
+# CHO other
+to_field 'cho_alternative', column('native_name')
```

Comment:
this should be `'cho_alternative_title'`

Reply:
that's what it was. I changed it because I couldn't find `cho_alternative_title` in https://github.com/sul-dlss/dlme/blob/master/docs/application_profile.md

Reply:
Should I add 'cho_alternative_title` to the application profile and change it here?

Reply:
Never mind - I was actually wrong here, and you were right - it is `cho_alternative`.